index.html

```diff
@@ -0,0 +1,125 @@
+      rel="stylesheet"
+    />
+    <title>Summit Event</title>
+  </head>
+  <body>
+    <header>
+      <nav class="desktop-menu desktop">
+        <div class="social desktop">
+          <ul class="social-list">
+            <li><a href="#" class="social-item"><img src="resources/images/facebook.png" class="desktop" alt="logo" id="facebook" /></a></li>
+            <li><a href="#" class="social-item"><img src="resources/images/twitter.png" class="desktop" alt="logo"/></a></li>
+            <li><a href="#" class="social-item">Language</a></li>
+            <li><a href="#" class="social-item">My Page</a></li>
+            <li><a href="#" class="social-item">Logout</a></li>
+          </ul>
+        </div>
+        <ul class="menu-list">
+          <li id="start-logo"><a href="#start" class="main-logo"><img src="resources/images/logo.png" class="desktop" alt="logo"/></a></li>
+          <li><a class="menu-item" href="#start">Hello</a></li>
+          <li><a class="menu-item" href="about.html"> About </a></li>
+          <li><a class="menu-item" href="#events"> Events </a></li>
+          <li>
+            <a class="menu-item" href="#speakers"> Speakers </a>
+          </li>
+          <li><a class="menu-item" href="about.html#contact"> Contact </a></li>
+          <li><button class="campaign">GB Cattle Bazaar</button></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="Hello" id="start">
+      <nav class="mobile menu">
+        <span class="hamburger"
+          ><img
+            src="resources/images/hambuger.svg"
+            alt="menu icon"
+            id="hamburger-icon"
+        /></span>
+        <div class="mobile-menu mobile">
+          <span class="xIcon"
+            ><img src="resources/images/Union-2.svg" alt="x icon"
+          /></span>
+          <ul class="mobile">
+            <li><a href="#hello">Hello</a></li>
+            <li><a href="about.html"> About </a></li>
+            <li><a href="#events"> Events </a></li>
+            <li><a href="#speakers"> Speakers </a></li>
+            <li><a href="about.html#contact"> Contact </a></li>
+          </ul>
+        </div>
+      </nav>
+      <span id="hello" class="hello">"Hello! Beef-and-dairyans"</span>
+      <h1 class="summit-name">GREAT BRITISH CATTLE BAZAAR <br />2027</h1>
+      <p class="summit-description">
+        The world's largest livestock sale, with over 40,000 cattle sold annually, as well as 200 million litres of milk. And including a sparkling lineup of the biggest names in the production of beef animals and dairy herds.
+      </p>
+      <div class="date-location">
+        <h2 class="date">2027.10.12(THU) ~ 13(FRI)</h2>
+        <p class="location">
+          @ National Museum of Cattle, Beef Center and more
+        </p>
+      </div>
+    </section>
+    <section class="main-program" id="events">
```

Comment:
- [ ] Please improve this section in order to match more the given design, change the height.

Yours:


![Screenshot from 2021-11-19 20-00-55](https://user-images.githubusercontent.com/62705947/142677145-7ef92789-44ca-4f5a-bc2d-db0e240ca882.png)
Original:

![Screenshot from 2021-11-19 20-00-56](https://user-images.githubusercontent.com/62705947/142677162-69c91323-6ebf-4c2e-9ea8-2d2c541f7f51.png)

Reply:
I use more text, but I fixed the style. So the size can be still a little different.

Reply:
- [ ] In order to finish this adjustment, you can try using `min-height: 200px;` to delimitate the height for all the cards. Or you can find another way if you wish. 😉